#!/bin/bash -l
#SBATCH -A ngi2016003
#SBATCH -p node
#SBATCH -n 8
#SBATCH -t 10-00:00:00
#SBATCH -J CombineGVCFs_batch2_interval0049
#SBATCH -o /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/std_out/CombineGVCFs_batch2_interval0049.out
#SBATCH -e /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/std_err/CombineGVCFs_batch2_interval0049.err
module load bioinfo-tools
module load GATK/3.5.0
java -Xmx200g -jar /sw/apps/bioinfo/GATK/3.5.0/GenomeAnalysisTK.jar -T CombineGVCFs \
-R /sw/data/uppnex/reference/biodata/GATK/ftp.broadinstitute.org/bundle/2.8/b37/human_g1k_v37.fasta \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418716.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34512400.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684521.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_260.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_142.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915555.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_281.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34741217.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645700.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35001249.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672302.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420989.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35000296.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/WGS-114.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34920479.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34913828.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645646.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34982684.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34629765.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002772.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_129.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34566927.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_173.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684562.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34833107.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34722837.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672217.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_266.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_109.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_305.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_151.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420342.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_176.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645774.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_307.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_241.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915236.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672380.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34998467.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832884.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030345.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34914765.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_267.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_127.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_156.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683998.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35000382.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915941.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_285.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34631211.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34631323.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_155.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917227.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34981425.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915318.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_173.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB40177793.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_185.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34845290.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_278.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34631169.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_134.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030300.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_115.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915726.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_178.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645620.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_159.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418777.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915539.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_172.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_229.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_272.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684030.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_189.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_224.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_224.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684643.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_232.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_137.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684614.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34920508.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_161.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_141.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_179.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_159.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_308.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_172.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915445.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2006010874.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_164.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684535.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_191.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672306.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34920519.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917219.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34914794.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_182.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34566662.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/WGS-132.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915353.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832874.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/KA06-0317.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_130.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34629777.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34646439.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34845218.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684593.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684642.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_151.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_177.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_235.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418855.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_128.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_178.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34722857.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418737.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_166.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672249.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_221.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002697.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_120.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_158.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420993.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35001223.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/KA06-0514.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917166.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_250.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2005007258.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684465.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030547.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_287.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030651.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915381.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_205.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34631384.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_280.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34998502.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915753.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_190.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_313.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839640.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_191.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_205.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_162.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2004001094.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_143.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_323.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35024059.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915189.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34738519.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34722885.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_163.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420270.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_216.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_144.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915982.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_148.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_179.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418854.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030683.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672188.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030560.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420803.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418740.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_164.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_264.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418821.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34566637.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_336.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_152.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_201.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35024065.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_183.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420762.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_136.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_130.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_182.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_120.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_168.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_107.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030697.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_132.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832849.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915229.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683648.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_170.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418862.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832866.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34566586.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_232.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34811343.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/WGS-10.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_142.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672523.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418896.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_159.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915257.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_209.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34738349.clean.dedup.recal.bam.genomic.vcf.gz \
 -L /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/00_intervals/interval0049.intervals \
-o /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/VCF/SweGen_batch2_interval0049.g.vcf.gz

