#!/bin/bash -l
#SBATCH -A ngi2016003
#SBATCH -p node
#SBATCH -n 8
#SBATCH -t 10-00:00:00
#SBATCH -J CombineGVCFs_batch1_interval0013
#SBATCH -o /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/std_out/CombineGVCFs_batch1_interval0013.out
#SBATCH -e /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/std_err/CombineGVCFs_batch1_interval0013.err
module load bioinfo-tools
module load GATK/3.5.0
java -Xmx200g -jar /sw/apps/bioinfo/GATK/3.5.0/GenomeAnalysisTK.jar -T CombineGVCFs \
-R /sw/data/uppnex/reference/biodata/GATK/ftp.broadinstitute.org/bundle/2.8/b37/human_g1k_v37.fasta \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_236.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418831.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_161.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915226.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_188.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_335.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_111.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_207.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_213.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35004118.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_123.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_108.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_133.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_194.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_122.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34833163.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030413.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003283.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_203.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_116.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917122.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34845223.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34833174.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839541.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_235.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34811342.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34742230.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33409865.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/WGS-40.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_103.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34737891.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_270.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_213.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_303.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_270.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420807.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002714.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_117.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030478.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_209.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418884.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915163.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030511.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418763.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32673342.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_273.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_213.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34628552.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35000324.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915554.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34920505.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672856.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_214.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003363.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35004109.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_124.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915683.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_181.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_320.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_134.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_131.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_163.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_342.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2004002406.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2004002226.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34916033.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_151.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_155.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_180.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_109.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_171.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672336.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030460.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839552.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003419.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_260.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34998496.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34998547.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB40176493.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_132.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030349.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34635592.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_208.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_192.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_133.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_139.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_217.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34914711.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_150.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_312.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_170.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_165.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030430.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_236.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_295.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915387.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684013.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_142.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420788.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030270.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030707.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_171.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_304.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418768.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32673355.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839607.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_226.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34646309.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_108.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_123.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003675.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_244.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_122.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672280.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917452.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_145.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_184.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/KA09-155.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915457.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_238.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030412.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34635651.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34631319.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003404.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_280.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915904.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_156.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_113.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_169.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_273.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_128.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_281.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_191.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915230.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_118.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34741181.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_171.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34512453.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_246.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_181.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_104.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_237.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_173.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34635581.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34628176.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672146.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_219.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34982678.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_279.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_149.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_264.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_140.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030461.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB40177843.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_176.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832877.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_288.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34998517.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_122.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_148.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34629731.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_328.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_145.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34512449.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2004001964.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34511586.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34738535.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_253.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915337.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_318.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34631314.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_152.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_226.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34920486.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915740.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34833169.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34914747.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915321.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_175.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_177.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32673472.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2006010895.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_259.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2006011891.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_113.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34742211.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2005005999.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_222.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_145.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_146.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35032758.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34913825.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34566686.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645724.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35001203.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34646205.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_126.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_341.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645632.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_116.clean.dedup.recal.bam.genomic.vcf.gz \
 -L /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/00_intervals/interval0013.intervals \
-o /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/VCF/SweGen_batch1_interval0013.g.vcf.gz

