#!/bin/bash -l
#SBATCH -A ngi2016003
#SBATCH -p node
#SBATCH -n 8
#SBATCH -t 2-00:00:00
#SBATCH -J CombineGVCFs_batch3_interval0004
#SBATCH -o /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_26/01_CombineGVCFs/std_out/CombineGVCFs_batch3_interval0004.out
#SBATCH -e /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_26/01_CombineGVCFs/std_err/CombineGVCFs_batch3_interval0004.err
module load bioinfo-tools
module load GATK/3.5.0
java -Xmx200g -jar /sw/apps/bioinfo/GATK/3.5.0/GenomeAnalysisTK.jar -T CombineGVCFs \
-R /sw/data/uppnex/reference/biodata/GATK/ftp.broadinstitute.org/bundle/2.8/b37/human_g1k_v37.fasta \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_150.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420270.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_169.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_157.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_224.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684629.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420762.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_174.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420953.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34566686.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_156.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_130.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_135.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_235.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_223.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915539.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_192.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_105.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002682.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_292.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684643.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_290.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672928.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34738282.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_194.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418854.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_244.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_239.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832874.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030460.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_190.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_170.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832902.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_206.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003633.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_111.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_170.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2004001964.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_201.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32685371.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030413.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_123.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915236.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32673355.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915213.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684058.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34913759.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2004001094.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/KA06-0317.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34920508.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645731.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34566662.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34629765.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_109.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_235.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_148.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030349.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_144.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_211.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_254.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/WGS-432.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35000324.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2006017217.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915755.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672146.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_194.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420303.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_142.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839640.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420286.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_177.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420788.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_311.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839648.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_336.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_141.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_142.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB40176493.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_141.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672249.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_180.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917434.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683439.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_220.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34566927.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_144.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32673472.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683645.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915709.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35001333.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_193.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_238.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_144.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_118.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672523.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917219.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418740.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_108.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915216.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_331.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_140.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34811388.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420766.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_178.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_157.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645700.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_335.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_189.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_134.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32673411.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_116.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_281.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34845260.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_262.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683674.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_154.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_226.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418731.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34998502.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32673358.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_266.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_112.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34738535.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_134.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_286.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_216.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915364.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_201.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_213.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030480.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003295.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34833150.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917188.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_184.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_114.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_134.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_112.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418841.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645724.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34635651.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915504.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839653.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_162.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_149.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_212.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002681.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/WGS-114.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_303.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_285.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645774.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672452.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_189.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_217.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_215.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839667.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_281.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_133.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_137.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_288.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917394.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002747.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34914708.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34742211.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_273.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35032758.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35024065.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002621.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420792.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_294.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_242.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915371.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_186.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672490.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB40176376.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_175.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_175.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915753.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002714.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35000332.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_259.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_184.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_101.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030651.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418831.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_259.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_284.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_291.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915318.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645646.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839729.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_312.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34722814.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_175.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34982679.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_101.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34631384.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34914747.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683451.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839724.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002706.clean.dedup.recal.bam.genomic.vcf.gz \
 -L /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_26/00_intervals/interval0004.intervals \
-o /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_26/01_CombineGVCFs/VCF/SweGen_batch3_interval0004.g.vcf.gz

