#!/bin/bash -l
#SBATCH -A ngi2016003
#SBATCH -p node
#SBATCH -n 8
#SBATCH -t 10-00:00:00
#SBATCH -J CombineGVCFs_batch5_interval0049
#SBATCH -o /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/std_out/CombineGVCFs_batch5_interval0049.out
#SBATCH -e /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/std_err/CombineGVCFs_batch5_interval0049.err
module load bioinfo-tools
module load GATK/3.5.0
java -Xmx200g -jar /sw/apps/bioinfo/GATK/3.5.0/GenomeAnalysisTK.jar -T CombineGVCFs \
-R /sw/data/uppnex/reference/biodata/GATK/ftp.broadinstitute.org/bundle/2.8/b37/human_g1k_v37.fasta \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_153.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672220.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34738282.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_183.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030516.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683645.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_127.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_137.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_146.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683439.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_251.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839648.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002707.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34722814.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_340.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_119.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34738327.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_257.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_204.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_231.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_190.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915696.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645731.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_125.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030463.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34811315.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB40176405.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34920560.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34737952.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_201.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB40176169.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915892.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_239.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_117.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_290.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2004002460.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_150.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839643.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33421010.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_144.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_101.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_343.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_170.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003681.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_242.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683451.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_103.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_276.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_109.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34840248.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_154.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_175.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_262.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832793.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_111.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_184.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34920555.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_322.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34722856.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003398.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030374.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_124.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_119.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839653.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_222.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030388.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_109.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003299.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_185.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_103.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34638938.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915444.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_120.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_292.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_115.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_134.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_147.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_286.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_101.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_196.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_246.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_137.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_285.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34840085.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915425.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832903.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34511593.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34722830.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_182.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_186.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34459793.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_180.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/WGS-69.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_154.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_205.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_215.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34461244.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_329.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/KA06-0328.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_175.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34840250.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34629769.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684632.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_127.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_230.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_266.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/WGS-369.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_315.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34459817.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_192.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_206.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915709.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_135.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_291.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_112.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_145.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_210.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418899.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003410.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_190.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_212.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_201.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_165.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_215.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_106.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420953.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_124.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002621.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684502.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003633.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915510.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34833150.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_219.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917271.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002747.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_147.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34833145.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_146.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_325.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_193.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839620.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_136.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_141.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915785.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002682.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_195.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839694.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34722842.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_310.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34982704.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418771.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683609.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35004483.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030709.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_187.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34638251.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34512398.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_245.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_170.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34722805.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34638362.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_149.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917394.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_105.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_125.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_287.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_113.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34737916.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832863.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34512432.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917031.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34638329.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_162.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_165.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_106.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_127.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_195.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_248.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_160.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030660.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672909.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32673372.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683674.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915743.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_314.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34913759.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_137.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003295.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_148.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030379.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_234.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_111.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34833186.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_196.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_233.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_174.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683769.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_294.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832902.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_269.clean.dedup.recal.bam.genomic.vcf.gz \
 -L /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/00_intervals/interval0049.intervals \
-o /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/VCF/SweGen_batch5_interval0049.g.vcf.gz

