#!/bin/bash -l
#SBATCH -A ngi2016003
#SBATCH -p node
#SBATCH -n 8
#SBATCH -t 10-00:00:00
#SBATCH -J CombineGVCFs_batch3_interval0001
#SBATCH -o /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/std_out/CombineGVCFs_batch3_interval0001.out
#SBATCH -e /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/std_err/CombineGVCFs_batch3_interval0001.err
module load bioinfo-tools
module load GATK/3.5.0
java -Xmx200g -jar /sw/apps/bioinfo/GATK/3.5.0/GenomeAnalysisTK.jar -T CombineGVCFs \
-R /sw/data/uppnex/reference/biodata/GATK/ftp.broadinstitute.org/bundle/2.8/b37/human_g1k_v37.fasta \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684456.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_124.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_115.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672539.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_215.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34629748.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_133.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_113.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35001299.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_333.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420792.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645676.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672240.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003438.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_141.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34741174.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672928.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_132.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030673.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003332.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915747.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_293.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34742257.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_141.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_169.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_204.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34811402.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839667.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917130.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_166.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_128.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_316.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32673411.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_134.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_254.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_104.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917413.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_249.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34512414.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915419.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35004485.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34920494.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672884.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_144.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34566693.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_216.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915371.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917229.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2006014195.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_282.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_189.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_255.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_223.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32684629.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418841.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_181.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2006012881.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34637539.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_249.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672452.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_108.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683630.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_176.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_119.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_309.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34737882.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_286.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_237.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34982679.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_123.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_195.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832731.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_135.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_255.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34629740.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_180.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915755.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB40176376.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_284.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420810.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420782.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_263.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_165.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_288.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_188.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_220.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_169.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_220.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34914708.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_186.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2006017217.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_239.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420766.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_203.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_172.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34629714.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35002681.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_121.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915823.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34638272.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_135.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_277.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_240.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_158.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35001295.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_167.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_184.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_211.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915511.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915771.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_179.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683601.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34629772.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_139.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_228.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2004001979.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34811339.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_243.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34629693.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_283.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915504.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35003266.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34461018.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_184.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_156.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_331.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420739.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_221.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34998516.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_174.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420818.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_114.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_183.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_302.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030501.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915213.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_125.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34461306.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_189.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_194.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_147.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_162.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34839664.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/KA06-0636.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35030369.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915386.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34915216.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34741180.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917267.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_188.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_118.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34917049.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672538.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_268.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34913796.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672350.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_178.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_252.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34742243.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_161.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683750.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_203.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_116.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_155.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_247.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_232.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33418878.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_136.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35000332.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_105.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/WGS-232.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34832769.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_179.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34639090.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_295.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB35024053.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_101.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_202.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_160.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_262.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_120.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420950.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_185.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_241.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_282.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672451.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34845260.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB33420307.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2656/piper_ngi/07_variant_calls/P2656_112.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_154.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34840288.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32672285.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB32683431.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P4302/piper_ngi/07_variant_calls/P4302_229.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/2004001187.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2652/piper_ngi/07_variant_calls/P2652_158.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34982640.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34645893.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016003/nobackup/NGI/ANALYSIS/P2655/piper_ngi/07_variant_calls/P2655_268.clean.dedup.recal.bam.genomic.vcf.gz \
-V /proj/ngi2016001/nobackup/NGI/ANALYSIS/OF-0839/piper_ngi/07_variant_calls/AB34638062.clean.dedup.recal.bam.genomic.vcf.gz \
 -L /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/00_intervals/interval0001.intervals \
-o /lupus/proj/ngi2016003/nobackup/vezzi/SwedishReferenceGenomeProjects/SwedishGenomeReference_16_09_20/01_CombineGVCFs/VCF/SweGen_batch3_interval0001.g.vcf.gz

